ts/Dashboards/EditMode/AccordeonMenu.ts

```diff
@@ -111,33 +121,82 @@ class AccordeonMenu {
         );
     }
 
+    /**
+     * Update the options object with new nested value, based on the property
+     * path. If the objects in the path are not defiend, the function will
+     * create them.
+     *
+     * @param propertyPath
+     * Path of the property for which the value should be updated.
+     * @param value
+     * New value of the property.
+     */
+    public updateOptions(
+        propertyPath: Array<string>,
+        value: boolean | string | number
+    ): void {
+        let currentLevel = this.changedOptions as any;
+        const pathLength = propertyPath.length - 1;
+
+        for (let i = 0; i < pathLength; i++) {
+            const key = propertyPath[i];
+
+            if (!currentLevel[key]) {
+                currentLevel[key] = {};
+            }
+
+            currentLevel = currentLevel[key];
+        }
+
+        currentLevel[propertyPath[pathLength]] = value;
+    }
+
+    /**
+     * Renders either a basic or nested element. This function can be recursivly
+     * called, if there are multiple nested options.
+     *
+     * @param options
+     * Configuration object of the Component options.
+     * @param parentNode the HTML to which the accoreon menu should be rendered to.
```

Comment:
Its a little bit unclear IMO. 
The parentNode is a container where the accordion is placed/rendered.